lib/plugins/fs-routes/routes-template.js

```diff
@@ -0,0 +1,113 @@
+const path = require('path')
+const fs = require('fs-extra')
+const sortBy = require('lodash.sortby')
+
+module.exports = async (api, pagesDir) => {
+  const routes = await collectRoutes({
+    dir: api.resolveBaseDir(pagesDir),
+    componentPrefix: path.join('#base/', pagesDir),
+    parentPath: '/'
+  })
+
+  return `export default ${renderRoutes(routes)}`
+}
+
+function renderRoutes(routes) {
```

Comment:
if you have time, we can start adding some unit tests at `test/unit` using `jest`, otherwise I'll do it later.

Reply:
@egoist I added tests.